workers/nomad-job-specs/smasher.nomad.tpl

```diff
@@ -65,9 +65,9 @@ job "SMASHER" {
       # The resources the job will require.
       resources {
         # CPU is in AWS's CPU units.
-        cpu = 1024
+        cpu = 2048
         # Memory is in MB of RAM.
-        memory = 12288
+        memory = 24576
```

Comment:
This box has 32GB of RAM and it only runs these jobs. Is there any reason to not put this number closer to the total RAM for the instance? With these changes we can't run more than one job at a time anyway.

Reply:
We can raise it a little bit, but there should be some overhead for the nomad client